src/psyclone/dynamo0p3_builtins.py

```diff
@@ -595,13 +595,29 @@ def __str__(self):
     def gen_code(self, parent):
         from psyclone.f2pygen import AssignGen
         # In this case we're raising each element of a field to a
```

Comment:
Please document the interface to this routine (sorry; I know you've barely touched it!)

Reply:
Documented interface to all "def gen_code" routines in the file.